src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator is now working");
```

Comment:
The message in `doWork` method should be more informative and indicate the type of machine. Consider specifying that an `Excavator` is working, to make the message clear.